Remove test apps not building with ppc and PTEST

For mpc8316-rdb in none/tests/ppc32, the oe compiler options are
inconsistent with the imposed test compiler options generating errors
as follows:
test_isa_2_07_part1
test_isa_2_07_part2
test_tm
test_touch_tm
    : unrecognized command line option '-mhtm'

jm-insns
testVMX
    : AltiVec not supported in this target

For the following tests, their inline assembler is inconsistent with
the ppce300c3 variant:
round.c:393
power5+_round.c:98
    : impossible constraint in 'asm'

For the following tests, with both mpc8316-rdb and with qemuppc bsp,
the inline assember is inconsistent with the oe compiler machine
tuning:
In memcheck/tests/ppc32
 power_ISA2_05.c:56
In none/tests/ppc32
 test_dfp1.c:85
 test_dfp2.c:160
 test_dfp3.c:157
 test_dfp4.c:73
 test_dfp5.c:73
    : impossible constraint in 'asm'

Upstream-Status: Pending

Signed-off-by: Dave Lerner <dave.lerner@windriver.com>

Index: valgrind-3.10.0/memcheck/tests/ppc32/Makefile.am
===================================================================
--- valgrind-3.10.0.orig/memcheck/tests/ppc32/Makefile.am
+++ valgrind-3.10.0/memcheck/tests/ppc32/Makefile.am
@@ -7,8 +7,7 @@ EXTRA_DIST = $(noinst_SCRIPTS) \
 	power_ISA2_05.stderr.exp power_ISA2_05.stdout.exp power_ISA2_05.vgtest \
 	power_ISA2_05.stdout.exp_Without_FPPO
 
-check_PROGRAMS = \
-	power_ISA2_05
+check_PROGRAMS = 
 
 power_ISA2_05_CFLAGS = $(AM_CFLAGS) $(WERROR) -Winline -Wall -Wshadow -g \
 		-I$(top_srcdir)/include @FLAG_M32@
Index: valgrind-3.10.0/none/tests/ppc32/Makefile.am
===================================================================
--- valgrind-3.10.0.orig/none/tests/ppc32/Makefile.am
+++ valgrind-3.10.0/none/tests/ppc32/Makefile.am
@@ -52,16 +52,8 @@ check_PROGRAMS = \
 	allexec \
 	bug129390-ppc32 \
 	bug139050-ppc32 \
-	ldstrev lsw jm-insns mftocrf mcrfs round test_fx test_gx \
-	testVMX twi tw xlc_dbl_u32 power5+_round power6_bcmp \
-	test_isa_2_06_part1 \
-	test_isa_2_06_part2 \
-	test_isa_2_06_part3 \
-	test_dfp1 test_dfp2 test_dfp3 test_dfp4 test_dfp5 \
-	test_isa_2_07_part1 \
-	test_isa_2_07_part2 \
-	test_tm \
-	test_touch_tm \
+	ldstrev lsw mftocrf mcrfs test_fx test_gx \
+	twi tw xlc_dbl_u32 power6_bcmp \
 	ldst_multiple \
 	data-cache-instructions
 
